File: app/src/main/java/de/bornholdtlee/defaultproject/ui/main/MainActivity.java
package de.bornholdtlee.defaultproject.ui.main;

import android.os.Bundle;

import de.bornholdtlee.defaultproject.R;
import de.bornholdtlee.defaultproject.ui.BaseActivity;
import de.bornholdtlee.defaultproject.utils.HockeyAppUtils;

public class MainActivity extends BaseActivity {

    public static final int CONTAINER = R.id.activity_main_container;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        showFragment(new MainFragment(), getSupportFragmentManager(), CONTAINER, true);
    }

    @Override
    protected void onResume() {
        super.onResume();

        /**
         * Info:
         * Diese Abfrage sollte nicht in der Splashscreen-Activity ausgeführt werden,
         * sondern erst in der darauf folgenden Activity
         */
        HockeyAppUtils.checkForCrashes(this);
    }
}

File: app/src/main/java/de/bornholdtlee/defaultproject/ui/main/MainActivity.kt
package de.bornholdtlee.defaultproject.ui.main

import android.os.Bundle
import de.bornholdtlee.defaultproject.R
import de.bornholdtlee.defaultproject.TAB_HOME
import de.bornholdtlee.defaultproject.TAB_MAP
import de.bornholdtlee.defaultproject.base.NavigationBaseActivity
import de.bornholdtlee.defaultproject.ui.map.MapFragment
import de.bornholdtlee.defaultproject.utils.HockeyAppUtils

class MainActivity : NavigationBaseActivity() {

    override val bottomMenuLayout: Int = R.menu.navigation_menu

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        onTabSelected(TAB_HOME, false)
    }

    override fun onResume() {
        super.onResume()
        HockeyAppUtils.checkForCrashes(this)
    }

    override fun onTabSelected(position: Int, wasSelected: Boolean): Boolean {
        if (!wasSelected) {
            when (position) {
                TAB_MAP -> showFragment(MainFragment.createInstance())
                TAB_HOME -> showFragment(MapFragment.createInstance())
                else ->
                    return false
            }
            return true
        }
        return false
    }
}